README.md

```diff
@@ -1,4 +1,223 @@
-#Shooter-Log-FrontEnd
```

Comment:
👍 
💯 Hands down best readme in codefellows history
Im proud and will use this as a reference for all of my future students :)

Increadible work